app/views/shared/menu/_large_menu.html.haml

```diff
@@ -37,6 +37,12 @@
               = t 'label_about'
 
     %ul.right
+      %li.powered-by
+        %img{src: '/favicon.ico'}
+        %span
+          = t 'powered_by'
+          %a{href: '/'}
```

Comment:
This was a last minute addition at Lynne's request. The homepage can be embedded, yes. It's probably best if users don't switch between embedded and non-embedded pages in a single session, but it should be ok.